docs/src/pages/[platform]/connected-components/authenticator/headless/advanced/example.react-native.mdx

```diff
@@ -0,0 +1,37 @@
+```js{9,13}
+import { Amplify } from 'aws-amplify';
+import { Authenticator, useAuthenticator } from '@aws-amplify/ui-react';
```

Comment:
Should be `@aws-amplify/ui-react-native`?

Reply:
ah thanks for catching that